libs/src/api/user.js

```diff
@@ -424,6 +424,9 @@ class Users extends Base {
     // Upload new metadata object to CN
     const { metadataMultihash, metadataFileUUID } = await this.creatorNode.uploadCreatorContent(newMetadata, updateEndpointTxBlockNumber)
 
+    // Make non-awaited /ipfs requests to all replicas for CID to warm cache before writing to chain
+    Utils.CIDHeadRequests(metadataMultihash, newMetadata.creator_node_endpoint.split(','))
```

Comment:
only thing i can think of is maybe somehow creator_node_endpoint string is null and this throws. not sure if a reality, but would not want to throw probably. perhaps you can make this 

Utils.makeCIDHeadRequests(metadatMulti, newMetadata.creator_node_endpoint)

and do the null check & split later

Reply:
agreed, done